runtime/altair/Cargo.toml

```diff
@@ -382,29 +425,28 @@ try-runtime = [
   "pallet-preimage/try-runtime",
   "pallet-proxy/try-runtime",
   "pallet-restricted-tokens/try-runtime",
-  "pallet-restricted-tokens/try-runtime",
   "pallet-rewards/try-runtime",
   "pallet-scheduler/try-runtime",
   "pallet-session/try-runtime",
+  "pallet-sudo/try-runtime",
   "pallet-timestamp/try-runtime",
   "pallet-transaction-payment/try-runtime",
+  "pallet-transfer-allowlist/try-runtime",
   "pallet-treasury/try-runtime",
   "pallet-uniques/try-runtime",
   "pallet-utility/try-runtime",
   "pallet-vesting/try-runtime",
-  "pallet-xcm-transactor/try-runtime",
   "pallet-xcm/try-runtime",
+  "pallet-xcm-transactor/try-runtime",
   "parachain-info/try-runtime",
-  "polkadot-runtime-common/try-runtime",
-  "runtime-common/try-runtime",
-  "sp-runtime/try-runtime",
 ]
 
 # A feature that should be enabled when the runtime should be build for on-chain
 # deployment. This will disable stuff that shouldn't be part of the on-chain wasm
 # to make it smaller like logging for example.
 on-chain-release-build = [
   "sp-api/disable-logging",
+  "runtime-common/on-chain-release-build",
```

Comment:
Can you give some details about this addition?

Reply:
Yes, I understand that if we compile altair with `on-chain-release-build` we would want the runtime common used to have that related feature also.